libkbfs/interfaces.go

```diff
@@ -952,6 +974,12 @@ type BlockOps interface {
 	Archive(ctx context.Context, tlfID TlfID, ptrs []BlockPointer) error
 }
 
+// AuthTokenRefreshHandler defines a callback to be called when an auth token refresh
+// is needed.
+type AuthTokenRefreshHandler interface {
```

Comment:
Can't we use `kbfscrypto.AuthTokenRefreshHandler` anywhere this is used (if it's even used anywhere anymore)?